packages/react-components/src/atoms/CookieButton.tsx

```diff
@@ -0,0 +1,41 @@
+import { css } from '@emotion/react';
+import React from 'react';
+import { neutral200 } from '../colors';
+import { cookieIcon } from '../icons';
+
+type CookieButtonProps = {
+  toggleCookieModal: () => void;
+  isOnboardable?: boolean;
+};
+
+const iconStyles = css({
+  display: 'flex',
+  position: 'fixed',
+  justifyContent: 'center',
+  alignItems: 'center',
+  padding: '0.5em',
+  backgroundColor: neutral200.rgb,
+  borderRadius: '4px',
+  left: '1em',
+  cursor: 'pointer',
+  border: `1.5px solid rgba(223, 229, 234, 0.3)`,
+  bottom: '1em',
+  '&.is-not-onboarded': {
+    bottom: '7em',
+  },
+});
+
+const CookieButton: React.FC<CookieButtonProps> = ({
+  toggleCookieModal,
+  isOnboardable,
+}) => (
+  <span
+    className={isOnboardable ? 'is-not-onboarded' : ''}
```

Comment:
This behaviour is specific to an app making the component less independent.
My suggestion would be to detach the positioning from the button itself.